File: desafio_flutter/lib/app/classes/usuario.dart
class Usuario {
  int? codigo;
  String? nome;
  String? cpf;
  String? endereco;
  String? telefone;

  Usuario({this.nome, this.cpf, this.endereco, this.telefone});

  Usuario.fromMap(Map map) {
    codigo = map['codigo'];
    nome = map['nome'];
    cpf = map['cpf'];
    endereco = map['endereco'];
    telefone = map['telefone'];
  }

  Map<String, Object?> toMap() {
    Map<String, dynamic>? map = {
      'nome': nome,
      'cpf': cpf,
      'endereco': endereco,
      'telefone': telefone,
    };
    if (codigo != null) {
      map['codigo'] = codigo;
    }
    return map;
  }

  @override
  String toString() {
    // TODO: implement toString
    return "id: $codigo Nome: $nome senha: $nome";
  }
}

File: desafio_sistema/desafio_sistema/Models/Usuario.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace desafio_sistema.Models
{
    public class Usuario
    {
        [Key]
        public int Codigo { get; set; }
        [Column]
        public string Nome { get; set; }
        public string Cpf { get; set; }
        public string Endereco { get; set; }
        public string Telefone { get; set; }
    }
}
